src/python/pants/backend/docker/util_rules/docker_binary.py

```diff
@@ -130,6 +131,67 @@ def run_image(
         )
 
 
+async def _get_docker_tools_shims(
+    *,
+    tools: Sequence[str],
+    optional_tools: Sequence[str],
+    search_path: Sequence[str],
+    rationale: str,
+) -> BinaryShims:
+    all_binary_first_paths: list[BinaryPath] = []
+
+    if tools:
+        tools_requests = [
+            BinaryPathRequest(binary_name=binary_name, search_path=search_path)
+            for binary_name in tools
+        ]
+
+        tools_paths = await MultiGet(
+            Get(BinaryPaths, BinaryPathRequest, tools_request) for tools_request in tools_requests
+        )
+        print(f"tools_paths={tools_paths}")
```

Comment:
These prints are noise - presumably they were not intended to be checked in?

Reply:
Ugh, forgot to remove them. PR on the way.

Reply:
https://github.com/pantsbuild/pants/pull/21354